### ISSUE TITLE
Some labels do not fit in the bar

### ISSUE THREAD

**Issue by ycollet** [Houston4444/RaySession#34]
![image](https://user-images.githubusercontent.com/1255536/67506047-a70cd080-f68c-11e9-9283-b0ecede49be0.png)

The label do not fit in the bar.

I use RaySession-0.8.1 on Fedora 30.

**Comment by Houston4444** [comment]
humm, "Dossier de la session" and "Précédent..." should be icons, not labels.
These icons comes from system icon theme and there names are not provided by your icon theme.
I should provide icons if theme doesn't provide them in all cases.
Was it the case before ? I mean, using the same icon theme with previous version of RaySession ?

**Comment by ycollet** [comment]
With the previous version of RaySession, it was already the case.
I use XFCE on Fedora 30.
I was on KDE some months ago but due to a problem with KWin and real time kernel, I switched to XFCE.

**Comment by ycollet** [comment]
How do I get  these default name for the theme and for these icons ?

Swtich to french.

En fait ce n'est pas tout à fait kwin, mais plasmashell qui freeze.
Ca n'arrive que lorsque je tourne sur noyau temps réel et que j'ai un application qui tourne en temps réel. Ca arrive quasi systématiquement et le workaround que j'ai trouvé: tuer plasmashell puis le redémarrer. Ensuite, il ne freeze plus.
La trace que j'avais récupéré via gdb me menait dans du code qt.
Je vais être absent une semaine donc je ne pourrai pas donner de nouvelles infos avant.

**Comment by Houston4444** [comment]
I don't really know because I use kdialog --geticon to find system icons. Maybe you can install kdialog withou many plasma dependencies.

Bon ça me rassure un peu mon plasmashell ne freeze pas.

**Comment by ycollet** [comment]
OK, so, you use kdialog inside RaySession to get these icons.
When I install RaySession, kdialog is not installed. That's maybe the problem.
I will try with kdialog installed and see if it fixes the problem.

**Comment by Houston4444** [comment]
> OK, so, you use kdialog inside RaySession to get these icons.
No, not at all ! I use kdialog to find visually system icon names for myself. kdialog install won't fix anything in RaySession, sure.

**Comment by ycollet** [comment]
I switched back to kde and there are no problems on kde.
The problem only appears on Xfce.

**Comment by Houston4444** [comment]
Ok, not surprised.
I did install Xfce to test, but I have not this problem.
I'll try to fix it providing fallback icons everywhere it's needed, then I'll put a comment here in order you can test it on xfce.

**Comment by ycollet** [comment]
Can you list the xfce packages you have installed ?
I maybe miss a xfce icon package ...

**Comment by Houston4444** [comment]
Normally I think I fixed the bug.
please try new version v0.8.2 or git master, and close it if it is good.
Thanks

**Comment by BrunoVernay** [comment]
Using v0.8.3 on Fedora 32 on Gnome Wayland, there is a similar issue:

- "Stopped" does not fit, can only see "topped" (Maybe just align left, since losing the end is not a big issue)
- Also, the window title is "Python3"

![Screenshot from 2020-05-16 14-07-35](https://user-images.githubusercontent.com/132261/82119485-4277a700-977f-11ea-890f-f3982c42cdcc.png)

**Comment by Houston4444** [comment]
I've seen this truncated text problem under librazik, too. I'll try something for that.
But, on your screenshot, I see it's not the main problem.
The main problem is that all icons use the wrong color scheme (all icons used from raysession itself, not from system), that is quite strange. They should be darker because the colour scheme is light.

**Comment by Houston4444** [comment]
@BrunoVernay  serais-tu capable de debugger le problème de couleurs d'icônes si je te dis où ça se situe dans le code ?
Je veux dire par là, as-tu quelques notions de python ou tout du moins de programmation pour y jeter un oeil ?

Ça se trouve dans src/gui/main_window.py, ligne 186:
`dark = isDarkTheme(self)`

que se passe-t-il si remplace cette ligne par:
`dark=False`

Si tu es capable d'aller plus loin, la fonction isDarkTheme est dans src/gui/gui_tools.py , mais là c'est seulement si tu peux comprendre le code.

**Comment by BrunoVernay** [comment]
Yes, I can code et c'est intéressant. I will try to find some time to look at it.

**Comment by Houston4444** [comment]
Super, Thanks you very beaucoup !

**Comment by BrunoVernay** [comment]
With `dark=False` c'est immediately better
![image](https://user-images.githubusercontent.com/132261/83862965-f89d3380-a722-11ea-8a2e-6215820ca85d.png)
